mmv1/products/lustre/Instance.yaml

```diff
@@ -0,0 +1,127 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: Instance
+description: Description
+base_url: projects/{{project}}/locations/{{location}}/instances
+update_mask: true
+self_link: projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+create_url: projects/{{project}}/locations/{{location}}/instances?instanceId={{instance_id}}
+update_verb: PATCH
+id_format: projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+examples:
+  - name: lustre_instance_basic
+    primary_resource_id: 'instance'
+    vars:
+      name: 'instance'
+      network_name: 'network'
+      address_name: 'ipaddress'
+      filesystem_name: 'filesystem'
+      capacity_gib: 21000
```

Comment:
Values like this can be hardcoded (which it looks like you've already done actually)

Reply:
Done